torch/_inductor/ir.py

```diff
@@ -6769,7 +6769,7 @@ def get_index(self, name):
     def indexing_from_args(self, indices):
         index = [*itertools.chain.from_iterable(indices)]
         assert len(index) == len(self.var_ranges), (index, self.var_ranges)
-        assert all(v not in self.var_ranges for v in index)
+        # assert all(v not in self.var_ranges for v in index)
```

Comment:
@jansel Do I understand correctly that this assertion is added for error prevention purposes? We are trying to retrace the loop body passed to the codegen with the same prefix `z` in order to further split the loops. It works fine without this assertion. Just wanted to get our feedback how to support this usage properly. Thanks.

Reply:
This sounds like you might be converting from IRNode indices to kernel indices more than once.  This could lead to bugs, we should fix it.